libs/entitlement/qa/entitlement-test-plugin/src/main/java/org/elasticsearch/entitlement/qa/test/URLConnectionNetworkActions.java

```diff
@@ -218,4 +242,191 @@ static void baseUrlConnectionGetContentWithClasses() throws Exception {
     static void sunHttpConnectionGetContentWithClasses() throws Exception {
         withJdkHttpConnection(conn -> conn.getContent(new Class<?>[] { String.class }));
     }
+
+    @EntitlementTest(expectedAccess = PLUGINS)
+    static void sunFtpURLConnectionConnect() throws Exception {
+        withJdkFtpConnection(URLConnection::connect);
```

Comment:
These `with` methods are very cool.

Reply:
At least they make these tests nice and compact

Reply:
Super easy to review!